bin/ons_csv_to_ctb_json_main.py

```diff
@@ -185,18 +184,7 @@ def main():
     if error_count:
         logging.warning(f'{error_count} errors were encountered during processing')
 
-    # There is not a separate tables file for v9.2.0. Use the output_file_types list
-    # to determine which file types will be written.
-    output_file_types = [FILE_CONTENT_TYPE_DATASET, FILE_CONTENT_TYPE_SERVICE,
-                         FILE_CONTENT_TYPE_TABLES]
-
-    if args.cantabular_version == DEFAULT_CANTABULAR_VERSION:
-        logging.info(
-            f'Output files will be written in Cantabular {args.cantabular_version} format')
-
-    elif args.cantabular_version == CANTABULAR_V9_2_0:
-        output_file_types = [FILE_CONTENT_TYPE_DATASET, FILE_CONTENT_TYPE_SERVICE]
-        convert_json_to_ctb_v9_2_0(ctb_datasets, ctb_tables, service_metadata)
+    if args.cantabular_version in KNOWN_CANTABULAR_VERSIONS:
```

Comment:
Maybe clarify that even if we write our their version string verbatim (e.g. 10.1.0), this is actually the same as 10.2.2 format.

Reply:
I'll add a log like this:
```
t=2023-01-13 15:44:44,281 lvl=INFO msg=Output files will be written in Cantabular 10.2.2 format, which is compatible with all versions of Cantabular from 9.3.0 to 10.2.2
```